### PR TITLE
Britech-388 Created cloud formation template for concourse

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: dataCenter/concourseCFT.json [hunk 1]
```diff
@@ -0,0 +1,539 @@
...
+      "Type": "String",
+      "Description": "Concourse Password",
+      "NoEcho": true
+    },
+    "AccessKeyId": {
+      "Type": "String",
+      "Description": "AWS AccessId. Must have permission to read s3 bucket containing secret key.",
+      "NoEcho": true
+    },
+    "SecretKey": {
```
Comment (pallenSL) [inline dataCenter/concourseCFT.json]: "AWSSecretKey" for consistency ?

File: dataCenter/concourseCFT.json [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+        "Monitoring": "false",
+        "Tags": [
+          {
+            "Key": "Name",
+            "Value": "Concourse_worker_cft"
+          }
+        ],
+        "NetworkInterfaces": [
+          {
+            "AssociatePublicIpAddress": true,
```
Comment (ceejsmith) [inline dataCenter/concourseCFT.json]: Why do we want a public IP address for a worker?

File: dataCenter/concourseCFT.json [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+    "DHCPOptions": {
+      "Type": "AWS::EC2::DHCPOptions",
+      "Properties": {
+        "DomainName": "eu-west-1.compute.internal",
+        "DomainNameServers": [
+          "AmazonProvidedDNS"
+        ]
+      }
+    },
+    "rtb7931fb1d": {
```
Comment (ceejsmith) [inline dataCenter/concourseCFT.json]: This route table and the following one are both existing resources? The names aren't very informative!

File: dataCenter/concourseCFT.json [hunk 4]
```diff
@@ -0,0 +1,539 @@
...
+        },
+        "IpProtocol": "tcp",
+        "FromPort": "3376",
+        "ToPort": "3376",
+        "SourceSecurityGroupId": {
+          "Ref": "ConcourseSecurityGroup"
+        }
+      }
+    },
+    "egress3": {
```
Comment (ceejsmith) [inline dataCenter/concourseCFT.json]: egress3 and egress4 begin with a lower case letter and don't have very descriptive names. Should they be renamed?

**Comment by ceejsmith** [comment]
This looks really good. The scripting for instances isn't as bad as I thought it might be, but we still need to factor it out.
I had a couple of questions, and would be happy to :shipit: once they're resolved

**Pull request closed without merging**